From 7b61f24db0f7de3ae2d28cdafc983efd2fe46b65 Mon Sep 17 00:00:00 2001
From: Mike McGrath <mmcgrath@redhat.com>
Date: Tue, 13 Oct 2009 11:58:54 -0500
Subject: [PATCH 2/2] fixes the -p option

---
 client/sendProfile.py |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/client/sendProfile.py b/client/sendProfile.py
index ff3a0a9..dcf60a3 100644
--- a/client/sendProfile.py
+++ b/client/sendProfile.py
@@ -164,6 +164,8 @@ if opts.scanOnly:
 
 if not opts.autoSend:
     if opts.printOnly:
+        for line in profile.getProfile():
+            print line
         sys.exit(0)
 
     def inner_indent(text):
-- 
1.6.2.5

