--- client/scan.py.orig	2008-03-08 11:57:57.000000000 -0600
+++ client/scan.py	2008-11-11 09:22:30.000000000 -0600
@@ -17,7 +17,7 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA.
 
 import smolt
-import simplejson, urllib
+import simplejson, urllib, httplib
 from i18n import _
 import config
 
@@ -60,21 +60,27 @@
     devices.append('System/%s/%s' % ( urllib.quote(h.host.systemVendor), urllib.quote(h.host.systemModel) ))
     for dev in devices:
         searchDevices = "%s|%s" % (searchDevices, dev)
-    scanURL='%s/w/api.php?action=query&titles=%s&format=json' % (smoonURL, searchDevices)
+    scanURL='%s/smolt-w/api.php?action=query&titles=%s&format=json' % (smoonURL, searchDevices)
+    params='action=query&format=json&titles=%s' % searchDevices
     try:
-        r = simplejson.load(urllib.urlopen(scanURL))
+        #r = simplejson.load(urllib.urlopen(scanURL))
+        headers = {"Content-type": "application/x-www-form-urlencoded",
+            "Accept": "text/plain"}
+        conn = httplib.HTTPConnection("smolts.org:80")
+        conn.request("POST", '/smolt-w/api.php', params, headers)
+        r = simplejson.load(conn.getresponse())
     except ValueError:
         print "Could not wiki for errata!"
         return
     found = []
     for page in r['query']['pages']:
         try:
-            r['query']['pages'][page]['id']
-            found.append('\t%swiki/%s' % (smoonURL, r['query']['pages'][page]['title']))
+            if int(page) > 0:
+                found.append('\t%swiki/%s' % (smoonURL, r['query']['pages'][page]['title']))
         except KeyError:
             pass
     if found:
-        print _("\tErrata Found!")
+        print _("\tErrata Found!  Please visit: ")
         for f in found: print "\t%s" % f
     else:
         print _("\tNo errata found, if this machine is having issues please go to")
